src/components/renderers/controls/control.html

```diff
@@ -2,9 +2,12 @@
     <div>
         <label ng-if="vm.showLabel" for="{{vm.id}}">{{vm.label}}</label>
     </div>
-    <div ng-transclude>
-    </div>
-    <div>
-        <alert ng-repeat="alert in vm.alerts" type="{{alert.type}}" >{{alert.msg}}</alert>
+    <div ng-transclude uib-popover-template="'validation.html'" popover-enable="vm.alerts.length>0" popover-trigger="mouseenter" popover-title="Invalid value" popover-placement="bottom-left">
```

Comment:
Is this a template file? If so, I think you didn't include it in the commit. If not, pleas explain.